<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetInfo</class>
 <widget class="QMainWindow" name="GetInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GetInfo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/resources/get_info.png</normaloff>:/icons/resources/get_info.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="hSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeDirectories"/>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="vLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="vSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableView" name="tableMimeSizes">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>95</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScanAll"/>
   <addaction name="actionSaveSnapshot"/>
   <addaction name="separator"/>
   <addaction name="actionSwitchToBytes"/>
   <addaction name="actionSwitchToKBytes"/>
   <addaction name="actionSwitchToMBytes"/>
  </widget>
  <action name="actionSwitchToBytes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/div_b.png</normaloff>:/icons/resources/div_b.png</iconset>
   </property>
   <property name="text">
    <string>Switch to Bytes</string>
   </property>
  </action>
  <action name="actionSwitchToKBytes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/div_kb.png</normaloff>:/icons/resources/div_kb.png</iconset>
   </property>
   <property name="text">
    <string>Switch to KBytes</string>
   </property>
  </action>
  <action name="actionSwitchToMBytes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/div_mb.png</normaloff>:/icons/resources/div_mb.png</iconset>
   </property>
   <property name="text">
    <string>Switch to MBytes</string>
   </property>
  </action>
  <action name="actionSaveSnapshot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/save.png</normaloff>:/icons/resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save snapshot</string>
   </property>
   <property name="toolTip">
    <string>Save results to database</string>
   </property>
  </action>
  <action name="actionScanAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/scan_all.png</normaloff>:/icons/resources/scan_all.png</iconset>
   </property>
   <property name="text">
    <string>Scan All</string>
   </property>
   <property name="toolTip">
    <string>Scan all directories</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
